criu/files-reg.c

```diff
@@ -1260,7 +1263,12 @@ static int check_path_remap(struct fd_link *link, const struct fd_parms *parms,
 
 		if (errno == ENOENT) {
 			link_strip_deleted(link);
-			return dump_linked_remap(rpath + 1, plen - 1, ost, lfd, id, nsid);
+			ret = dump_linked_remap(rpath + 1, plen - 1, ost, lfd, id, nsid);
+			if (ret < 0 && errno == ENOENT) {
+				pr_info("Unable to link remap %s, fall back to dump_ghost_remap\n", rpath + 1);
```

Comment:
pls add a comment that explains why we need this code path

Reply:
ok, I will add it as soon as I solve the failed tests